frontend/src/queries/nodes/DataTable/DataTableExport.tsx

```diff
@@ -31,36 +36,46 @@ export async function startDownload(
     exportCall: (exportData: TriggerExportProps) => void,
     format: ExporterFormat = ExporterFormat.CSV
 ): Promise<void> {
+    const shouldOptimize = shouldOptimizeForExport(query)
+
+    let exportSource = query.source
+
+    // Remove person column from the source otherwise export fails when there's 1000+ records
```

Comment:
Feels like more of a workaround, which is fine for now, but what do we have to do to improve performance instead?

Reply:
i'm not sure, but this has been the behavior for a while, [since even before this PR](https://github.com/PostHog/posthog/pull/18046/files#diff-20102c00691ff801df897e2acc2c6a24ca5e4b3bf7afafcccf90732b3675296bR32-R36). but this was replacign it only on the export query, not on the source one

Reply:
the `person` column would be the distinct_id plus all their properties, which I don't think is suitable anyway for a csv/xls export. the better fix would be prompting the customer to how they want to identified the user I think